Description: Disable zerocopy-0_8 feature for we have only 0.7
 .. and the sole dependent, rkyv, not (yet) needing it
Forwarded: not-needed
Last-Update: 2025-01-17
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -50,3 +50,3 @@
 
-[dependencies.zerocopy]
+[disabled.dependencies.zerocopy]
 version = "0.8"
@@ -55,3 +55,3 @@
 
-[dependencies.zerocopy-derive]
+[disabled.dependencies.zerocopy-derive]
 version = "0.8"
@@ -62,5 +62,5 @@
 default = []
-zerocopy-0_8 = [
-    "dep:zerocopy",
-    "dep:zerocopy-derive",
-]
+#zerocopy-0_8 = [
+#    "dep:zerocopy",
+#    "dep:zerocopy-derive",
+#]
